x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx

```diff
@@ -337,14 +426,15 @@ export class MapsAppView extends React.Component {
         showSearchBar={true}
         showFilterBar={true}
         showDatePicker={true}
+        // @ts-ignore
```

Comment:
You can remove `@ts-ignore` comment by making the next line `showSaveQuery={!!getMapsCapabilities().saveQuery}` and convert getMapsCapabilities().saveQuery into a boolean

Reply:
@nreese When I removed the `@ts-ignore` new errors popped up on the [TopNavMenu](https://github.com/elastic/kibana/blob/9d13b6f5774ac29f230f146fc48f1914208865aa/x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx#L396) component call this was because `TopNavMenu` was not having the prop [onFiltersUpdated](https://github.com/elastic/kibana/blob/9d13b6f5774ac29f230f146fc48f1914208865aa/x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx#L409) and [onRefreshChange](https://github.com/elastic/kibana/blob/9d13b6f5774ac29f230f146fc48f1914208865aa/x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx#L414). My plan is to remove these props. Can you confirm whether it is ok?

Reply:
onFiltersUpdated and onRefreshChange should not be removed. Those are needed to handle changes to the filter bar and refresh configurations.

Reply:
Added them back. Also added those props to `TopNavMenu` component.